pkgs/by-name/ol/olympus/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts
+
+set -eu -o pipefail
+
+attr=olympus.passthru.unwrapped
```

Comment:
```suggestion
attr=olympus-unwrapped
```
Also, move `update.sh` to `pkgs/by-name/ol/olympus-unwrapped`.